assets/js/blocks/cart-checkout/cart/full-cart/cart-line-item-row.js

```diff
@@ -20,28 +20,93 @@ import ProductLowStockBadge from './product-low-stock-badge';
 /**
  * Cart line item table row component.
  */
-const CartLineItemRow = ( { lineItem = {} } ) => {
+const CartLineItemRow = ( {
+	lineItem = {
+		key: '',
+		id: 0,
+		quantity: 0,
+		name: '',
+		summary: '',
+		short_description: '',
+		sku: '',
+		remaining_stock: null,
+		low_stock_remaining: null,
+		low_stock_threshold: null,
+		sold_individually: false,
+		permalink: '',
+		images: [],
+		variation: [],
+		prices: {
+			currency_code: 'US',
+			currency_minor_unit: 2,
+			currency_symbol: '$',
+			currency_prefix: '$',
+			currency_suffix: '',
+			currency_decimal_separator: '.',
+			currency_thousand_separator: ',',
+			price: '0',
+			regular_price: '0',
+			sale_price: '0',
+			price_range: null,
+		},
+		totals: {
+			currency_code: 'US',
+			currency_minor_unit: 2,
+			currency_symbol: '$',
+			currency_prefix: '$',
+			currency_suffix: '',
+			currency_decimal_separator: '.',
+			currency_thousand_separator: ',',
+			line_subtotal: '0',
+			line_subtotal_tax: '0',
+			line_total: '0',
+			line_total_tax: '0',
+		},
+	},
+} ) => {
+	/**
+	 * @type {import('@woocommerce/type-defs/cart').CartItem}
+	 */
 	const {
-		key = '',
-		name = '',
-		summary = '',
-		permalink = '',
-		images = [],
-		variation = [],
-		prices = {},
+		name,
+		summary,
+		remaining_stock: remainingStock,
+		low_stock_threshold: lowStockThreshold,
+		low_stock_remaining: lowStockRemaining,
+		permalink,
+		images,
+		variation,
+		prices,
 	} = lineItem;
 
 	const {
 		quantity,
 		changeQuantity,
 		removeItem,
 		isPending: itemQuantityDisabled,
-	} = useStoreCartItemQuantity( key );
+	} = useStoreCartItemQuantity( lineItem );
 
-	const currency = getCurrency();
+	const currency = getCurrency( {
```

Comment:
These fields map 1:1; could we just pass through `prices` here?

Reply:
Oh I guess we could! The only reason I didn't was because `prices` includes more than just currency data and if we decide to have specific types for `getCurrencyData` that could result in red squigglies. Still with it currently accepting any object shouldn't be a problem.

Reply:
done.